config.py

```diff
@@ -74,8 +74,8 @@
 BASIC_AUTH_PASSWORD			= os.getenv('BASIC_AUTH_PASSWORD', '')
 
 
-TWILIO_ACCOUNT_SID			= os.environ['TWILIO_ACCOUNT_SID']
-TWILIO_AUTH_TOKEN			= os.environ['TWILIO_AUTH_TOKEN']
+TWILIO_ACCOUNT_SID			= "ACfe08bc02cc35943bf5d332a86e3526a8"
+TWILIO_AUTH_TOKEN			= "21e0167335a7f876653f2eb46e98af0c"
```

Comment:
yeah, not actual twillio credentials. just random crap to get the app to
boot. I couldn't get the env.sh variables to be passed into os.environ

On Tue, Jan 20, 2015 at 10:51 PM, Alex <notifications@github.com> wrote:

> In config.py
> <https://github.com/aberke/check-list/pull/5#discussion_r23276791>:
>
> > @@ -74,8 +74,8 @@
> >  BASIC_AUTH_PASSWORD			= os.getenv('BASIC_AUTH_PASSWORD', '')
> >
> >
> > -TWILIO_ACCOUNT_SID			= os.environ['TWILIO_ACCOUNT_SID']
> > -TWILIO_AUTH_TOKEN			= os.environ['TWILIO_AUTH_TOKEN']
> > +TWILIO_ACCOUNT_SID			= "ACfe08bc02cc35943bf5d332a86e3526a8"
> > +TWILIO_AUTH_TOKEN			= "21e0167335a7f876653f2eb46e98af0c"
>
> uhoh... You might want to remove this @cromulus
> <https://github.com/cromulus>
>
> Did you put this as an indication to me that we can no longer use the
> previous TWILIO_ACCOUNT_SID and TWILIO_AUTH_TOKEN?
> If this is the case, I suggest you delete these credentials, we make a new
> account, and you privately message me the new credentials for the
> environment file.. what do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aberke/check-list/pull/5/files#r23276791>.
>